### DISCUSSION TITLE
How to complete installing copilot: 'extension activation failed: "undefined" '

### DISCUSSION THREAD

**Discussion by go-bears** [community/community#9075] [Copilot Conversations]
Hello!

I'd like to know how to complete installation of GitHub Pilot's extension. I was invited to the technical preview and clicked the link and completed the installation. I wasn't behind a VPN or firewall when I installed the extension and I don't use one in my current setup. 

I've pasted the error message, but it's sufficiently vague that I don't know what else to do.  I've done the following:

install and enable the extension
authenticated via github
restarted vscode
I'm not behind a vpn or firewall

Thanks for any advice! Excited to be part of the technical preview community!

<img width="1349" alt="Screen Shot 2021-12-18 at 3 30 38 PM" src="https://user-images.githubusercontent.com/6887377/146658210-f6828ca1-4a40-4d8a-8a20-ecf6ef700771.png">

**Comment by sam-atkins** [comment]
Hi, I have something similar. In the browser it says Copilot is enabled in my account but in VSCode I get a very similar error to go-bears, just mine says user not authorized:

```
[INFO] [default] [2021-12-22T08:53:25.580Z] Experimentation service initialized
[INFO] [default] [2021-12-22T08:53:25.581Z] Stringified features: {}
[INFO] [default] [2021-12-22T08:53:25.581Z] Feature for copilotaa: false
[INFO] [default] [2021-12-22T08:53:25.581Z] Feature for copilotincludeprompt: none
[INFO] [default] [2021-12-22T08:53:25.581Z] Feature for copilotset1: undefined
[INFO] [default] [2021-12-22T08:53:25.581Z] Feature for copilotghgh: false
[INFO] [default] [2021-12-22T08:53:25.581Z] Feature for copilotfim: false
[INFO] [auth] [2021-12-22T08:53:25.984Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2021-12-22T08:53:25.984Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Edit**
The VS Code extension prompted me to authenticate with GitHub and now CoPilot is working for me.
